### PR TITLE
Update multi-key.md

### PR DESCRIPTION
**Pull request by paulosacramento**
This seems to be a common misunderstanding among beginners. @sbddesign also considers it relevant, so this is why I'm proposing adding this statement.

### CODE REVIEW

File: guide/private-key-management/multi-key.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ In the case of a personal wallet, one individual will control all the keys but h
 
 In the case of a shared wallet, different people will control the keys. The number of keys and required co-signers will depend on the use case. With spouses sharing a *joint account*, a simple 1-of-2 multi-key setup might suffice, meaning there are two keys but only one is required to sign for a transaction to be valid. At the other end of a spectrum a company might require a more complex 3-of-5 setup, requiring three of the five co-signers to approve any transaction.
 
+Since this is a common misunderstanding, we would like to remember that the usage of the multiple keys is necessary only for signing outgoing transactions, but not for receiving funds.
```
Comment (sbddesign) [inline guide/private-key-management/multi-key.md:41]:
```suggestion
It is important to remember that the usage of multiple keys is necessary only for signing outgoing transactions, but not for receiving funds. This is a common misunderstanding.
```

**Review by sbddesign — COMMENTED** [review]
I like your idea of adding this, but I think it should be worded in passive voice like the other content in the Guide. This is worded more in the active voice ("we would like..."

**Comment by GBKS** [comment]
While we're at it, we should review how we refer to public and private keys on this page altogether. I think I just spotted an error, as the page states that an application combines private keys into a multisig, while it's really the public keys. Could also be a new PR if you want to keep this one small.

File: guide/private-key-management/multi-key.md (hunk 1 above)
Comment (paulosacramento) [reply guide/private-key-management/multi-key.md:41]: You are absolutely right! Thank you very much @sbddesign .

**Comment by paulosacramento** [comment]
I would open a new PR for this second correction @GBKS .

**Comment by ConorOkus** [comment]
Although technically correct when we say "signing a transaction" it doesn't always feel that intuitive to me. Can we include the purpose is to send/spend the funds? You did allude to it by saying "outgoing".

**Comment by danielnordh** [comment]
Not sure a lot of people think they have to approve incoming transactions.
We could achieve the same clarifying effect just by adding _outgoing_.

"requiring three of the five co-signers to approve any _outgoing_ transaction."

**Comment by paulosacramento** [comment]
How would you guys rephrase @sbddesign version, @ConorOkus and @danielnordh ?

> It is important to remember that the usage of multiple keys is necessary only for signing outgoing transactions, but not for receiving funds. This is a common misunderstanding.

**Review by GBKS — APPROVED** [review]

**Pull request merged**